### DISCUSSION TITLE
wasm(pion) + webcodecs + webtransport

### DISCUSSION THREAD

**Discussion by byterldp** [pion/webrtc#2267] [Q&A]
we want build a rtc demo run in browser that not use webrtc.
1.transport part will use webtransport (unreliable datagram not stream)for rtp/rtcp send/recv, instead of ice
2.webcodecs for audio/video encode and decode 
3.rtp/rtcp packetizer/unpacktizer, lost recovery(NACK), BWE(bandwith estimation), fec, jitter buffer etc 

the 3 part need build wasm of exist module , is pion webrtc suitable for such feature ?  

thanks.

**Comment by Odinvt** [comment]
if someone is looking for this, the folks over at meetecho are experimenting with it : https://github.com/meetecho/imquic
